.github/workflows/master.yml

```diff
@@ -248,7 +248,7 @@ jobs:
       - name: Install dependencies
         run: |
           source ./dev/install-common-deps.sh
-          pip install pyspark
+          pip install pyspark "numpy<1.24.0"
```

Comment:
Can we avoid the pin here? I worry we'll forget to remove it. If these `models` tests are duplicating tests from cross version tests, we should deduplicate by removing them from `models` tests

Reply:
It's actually for mlflow evaluate and it's use of shap. I really like the idea of splitting that out into its own CI suite, though. We'd have to leave the pin in, but it wouldn't hamper our ability for the much larger scope of mlflow/models to be pinned to an old version of numpy. :) I'm going to push that change now.